lib/rubocop/cop/sorbet/signatures/signature_build_order.rb

```diff
@@ -37,13 +37,17 @@ class SignatureBuildOrder < SignatureCop
             :abstract,
             :override,
             :overridable,
+            :final,
```

Comment:
As per @paracycle in Slack:

> `final` is usually after `abstract`, `override`, `overridable`

Reply:
Actually, I think I was wrong in that message. First of all `abstract.final` does not make sense and `final.override` makes more sense than `override.final`. `final` has more signal, so probably should come first. WDYT @Morriar?